### ISSUE TITLE
Create a library of typical parsers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vlasovskikh/funcparserlib#6]
```
It would be nice to have a library of typical parsers, such as a parser of
int or float literals, escaped strings etc.

Typical tokenizer specs could be also useful.

```

Original issue reported on code.google.com by `andrey.vlasovskikh` on 6 Oct 2009 at 9:03
* Blocked on: #7

**Comment by vlasovskikh** [comment]
I no longer think it's a good idea.
